src/providers/v3/static-subgraph-provider.ts

```diff
@@ -110,6 +115,12 @@ const BASES_TO_CHECK_TRADES_AGAINST: ChainTokenList = {
     USDT_OPTIMISTIC_KOVAN,
     USDC_OPTIMISTIC_KOVAN,
   ],
+  [ChainId.POLYGON]: [USDC_POLYGON, WETH9[ChainId.POLYGON]!, WMATIC_POLYGON],
```

Comment:
hmmmmm, non-null assertions are forbidden in the interface, usually code smell, fyi.

but also the WETH on polygon is not a WETH9 contract, it's subtly different, it could cause bugs to bundle them, rather than bundling the wrapped native currency and having weth on polygon just be another token like dai